test/classes/initializers/records/where-clause4b.future

```diff
@@ -1,12 +0,0 @@
-error message: limit the candidate output
-
-At the moment, this test outputs an additional number of potential matches that
-were rejected.  All of these appear to be generated for iterators, and are
-likely not "proper initializers".  That is one problem.  This also makes it
-apparent that users which write an incorrect call to an initializer are likely
-to getting increasingly more verbose and unintelligible error messages as we
-convert our library types to use the new initializers.  Something should be done
-to limit the number of candidates we show in such error cases.
-
-When this is fixed, remove this version of the test and the prediff on
-where-clause4.chpl
```

Comment:
This says that the version of the test and the prediff on where-clause4 should be removed when the future is removed

Reply:
Good catch!